integration-tests/resteasy-reactive/src/main/java/io/quarkus/it/resteasy/server/GreetingResource.java

```diff
@@ -10,7 +10,7 @@ public class GreetingResource {
 
     @GET
     public Response hello() {
-        return Response.status(200).cookie(NewCookie.valueOf("Set-Cookie: greeting=hello; SameSite=Lax")).build();
+        return Response.status(200).cookie(NewCookie.valueOf("Set-Cookie: greeting=hello; SameSite=Lax;")).build();
```

Comment:
Hmmm. This worries me a bit. You don't trigger the issue if you don't have a trailing `;`?

Reply:
I've included more tests to check that the header doesn't have a trailing `;`, this won't raise any issues.